website/src/blogs/2017-02-28-using-openebs-to-build-a-true-on-premise-container-service.md

```diff
@@ -0,0 +1,53 @@
+
+Well, the troubling infrastructure piece to build the true container service in your enterprise is the persistent storage for your applications. No surprise there. VMs or bare metal can be deployed using well-known tools such as OpenStack and KVM, but for deploying EBS or GPD equivalent, you would need something like OpenEBS.
+
+OpenEBS helps the enterprises to build an AWS EBS equivalent or GPD equivalent platforms on-premise. OpenEBS is architected to ease the provisioning and management of persistent volumes at scale. It will have most of the elements of the popular AWS EBS. I had discussed the comparison between AWS EBS and OpenEBS with few community members in our last meetup. The slides are posted at [https://www.slideshare.net/OpenEBS/openebs-containerized-storage-for-containers-meetup-2](https://www.slideshare.net/OpenEBS/openebs-containerized-storage-for-containers-meetup-2)
+
+Containerization of block storage volumes gives the benefit of flexible storage upgrade schedules, treating the storage volume as part of your K8s POD, etc. However, there is more to consider as benefits of OpenEBS. OpenEBS is “Open source” EBS. A quick feature comparison is as follows
+
+![Comparing AWS EBS & OpenEBS](https://cdn-images-1.medium.com/max/800/1*uu_mIhdqobjf3ftNOtf8KQ.png)
+
+Next, provisioning and consuming persistent volumes through OpenEBS is very similar and simple to that of AWS EBS
+
+In AWS EBS a user creates and attaches a disk to an EC2 instance. The nuances of underlying storage protocols are hidden underneath and not exposed to the consumer/user.
+
+![Connectivity in AWS EBS](https://cdn-images-1.medium.com/max/800/1*zShnxODcXjTNu-X-qsJa5g.png)
+
+Flow of provisioning and consuming persistent block volumes on AWS EBS
+In OpenEBS, it is very similar. Once the OpenEBS volumes are expressed as intent in the application YAML config file, the volumes are automatically created on the OpenEBS platform, mounted on K8s minions, and the persistent storage is made available to the application.
+
+Example of OpenEBS volumes getting consumed through iSCSI:
+
+![Connectivity in OpenEBS](https://cdn-images-1.medium.com/max/800/1*Mh9MzX5a_YbV9K_LR8EynA.png)
+
+An example of OpenEBS volumes getting dynamically provisioned and consumed using k8s-openebs provider is shown below
+
+[https://github.com/openebs/openebs/blob/master/k8s-demo/my-nginx-pod-on-openebs.yaml](https://github.com/openebs/openebs/blob/master/k8s-demo/my-nginx-pod-on-openebs.yaml)
+
+The next comparison of OpenEBS to AWS EBS is how the volume snapshots are managed for data protection.
+
+![Snapshots on AWS EBS vs OpenEBS](https://cdn-images-1.medium.com/max/800/1*elAnAeYarCwxeCEyXv_Xow.png)
+**Comparing snapshots management on OpenEBS to that of AWS EBS**  
```

Comment:
please give a line space for better formatting (after line 44)

Reply:
It already has spacing
<img width="1440" alt="Screenshot 2021-07-06 at 1 28 12 PM" src="https://user-images.githubusercontent.com/34312966/124563842-0c532600-de5e-11eb-9b4e-08976aaa0620.png">

Reply:
I meant in the code as well 😄

Reply:
like line number 51, 52

Reply:
This is figure caption, so line space is not provided